dbschema/producible.esdl

```diff
@@ -4,7 +4,20 @@ module default {
       delegated constraint exclusive;
     };
     
-    scriptureReferences: multirange<int32>;
+    scripture: Scripture::Collection {
```

Comment:
Renamed so that queries can do something like
```edgeql
select Film {
  scriptureReferences := (select .scripture.verses {
    `start`: { book, chapter, verse },
    `end`: { book, chapter, verse }
  })
}
```
This free shape being different than a `Scripture::Collection` means the same field/column name cannot be used.